.github/workflows/release.yml

```
@@ -50,3 +50,65 @@ jobs:
           files: |
             pie.phar
             pie.phar.asc
+      - name: Store PHAR as artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: phar-binary
+          path: pie.phar
+
+  docker-binary-only-image:
+    needs: release-phar
+    name: Docker binary-only image
+    runs-on: ubuntu-latest
+    if: ${{ startsWith(github.ref, 'refs/tags/') }}
+
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Restore built PHAR
+        uses: actions/download-artifact@v4
+        with:
+          name: phar-binary
```

After a rebase, this can become:

```suggestion
      - name: Fetch built PHAR from artifacts
        uses: actions/download-artifact@v4
        with:
          name: pie-${{ github.sha }}.phar
      - name: Verify the PHAR
        env:
          GH_TOKEN: ${{ github.token }}
        run: gh attestation verify pie.phar --repo ${{ github.repository }}
```

---

This still needs to be added; please use the existing artifact (`pie-${{ github.sha }}.phar`), and add the `attestation verify` step

---

NM was just updated

---

Done in [e0c258d](https://github.com/php/pie/pull/122/commits/e0c258d53b3563af89687967013260de6aafff99).